h2o-py/tests/testdir_algos/automl/pyunit_automl_distributions.py

```diff
@@ -0,0 +1,144 @@
+        dict(response="binomial", distribution="bernoulli",
+             algos=['DRF', 'DeepLearning', 'GBM', 'GLM', 'StackedEnsemble', 'XGBoost'], max_models=12),
+        dict(response="quasibinomial", distribution="quasibinomial", algos=['GBM', 'GLM', 'StackedEnsemble'],
+              max_models=17, fail=True),  # needed to be able to build SE
+        dict(response="quasibinomial", distribution="fractionalbinomial", algos=['GLM'], fail=True),
+        dict(response="multinomial", distribution="multinomial",
+             algos=['DRF', 'DeepLearning', 'GBM', 'GLM', 'StackedEnsemble', 'XGBoost'], max_models=12),
+        dict(response="gaussian", distribution="gaussian",
+             algos=['DRF', 'DeepLearning', 'GBM', 'GLM', 'StackedEnsemble', 'XGBoost'], max_models=12),
+        dict(response="ordinal", distribution="poisson",
+             algos=['DeepLearning', 'GBM', 'GLM', 'StackedEnsemble', 'XGBoost'], nrows=400),
+        dict(response="gaussian", distribution="gamma",
+             algos=['DeepLearning', 'GBM', 'GLM', 'StackedEnsemble', 'XGBoost'], nrows=400),
+        dict(response="gaussian", distribution="laplace", algos=['DeepLearning', 'GBM']),
+        dict(response="gaussian", distribution=dict(distribution="quantile", quantile_alpha=0.25), algos=['DeepLearning', 'GBM']),
+        dict(response="gaussian", distribution=dict(distribution="huber", huber_alpha=.3),
+             algos=['DeepLearning', 'GBM'], max_models=12),
+        dict(response="gaussian", distribution=dict(distribution="tweedie", tweedie_power=1.5),
+             algos=['DeepLearning', 'GBM', 'GLM', 'StackedEnsemble', 'XGBoost']),
+        dict(response="ordinal_factors", distribution="ordinal", algos=[], fail=True),
+        dict(response="gaussian", distribution=dict(distribution="custom", custom_distribution_func="FILLED_LATER_IN_THE_TEST"), algos=["GBM"]),
+    ]
+    seed = 9803190
+
+    def make_data(size):
+        np.random.seed(seed=seed)
+        a = np.tile([0., 0.7333], int(size / 2))
+        a = a[np.argsort(np.random.uniform(size=size))]
+        b = 2. * a ** 0.1 + 3. + 8 * np.random.uniform(size=size)
+        d = np.tile([1, 2, 3, 4, 5, 6, 7, 8], int(size / 8))
+        c = 3 * d - 2 + np.random.uniform(size=size)
+        e = np.tile(["class 0", "class A"], int(size / 2))
+        f = np.tile(["class 0", "class A", "class alpha", "class aleph"], int(size / 4))
+
+        df = h2o.H2OFrame(pd.DataFrame(dict(quasibinomial=a, gaussian=b, noise=c, ordinal=d, binomial=e, multinomial=f)))
+        df["ordinal_factors"] = df["ordinal"].asfactor()
+        return df
+
+    def get_distribution(model_id):
+        ap = h2o.get_model(model_id).actual_params
+        if "metalearner_params" in ap.keys():
+            ap = h2o.get_model(model_id).metalearner().actual_params
+        distribution = ap.get("distribution", ap.get("family"))
+        if distribution == "binomial":
+            distribution = "bernoulli"
```

Comment:
curious: why do you need this additional conversion here? is it only because of GLM or sth?

Reply:
Yeah, we have a naming conflict between GLM which is based on `family` values of glmnet, but "bernoulli" is a better name for distribution.  More info: https://www.statology.org/bernoulli-vs-binomial/